mambaSharedFramework/Pantos-Generic HLS Playlist Parsing/Pantos-Generic Tag Validators/EXT_X_TARGETDURATIONLengthValidator.swift

```diff
@@ -31,11 +31,11 @@ class EXT_X_TARGETDURATIONLengthValidator: HLSPlaylistOneToManyValidator {
             
             guard let one = one,
                 let many = many,
-                let max: CMTime = one.value(forValueIdentifier: PantosValue.targetDurationSeconds)
+                let max: Double = one.value(forValueIdentifier: PantosValue.targetDurationSeconds)
                 else { return nil }
             
             for tag in many {
-                if tag.duration > max {
+                if tag.duration.seconds.rounded(.down) > max {
```

Comment:
this bit in the spec:
> when rounded to the nearest integer

Makes me think always rounding down is incorrect.

Reply:
Yeah, this is correct. Fixing...